extensions/txAdminClient/resource/cl_scaleform.lua

```diff
@@ -0,0 +1,89 @@
+-- Announce Scaleform to every players
+local announce_title = "Staff announce"  -- Set the title of the announcement here.
+local AnnouncementScaleform
+Scaleform = {}
+local scaleform = {}
+scaleform.__index = scaleform
+
+
+function StartDraw(time)
```

Comment:
Can I get a good reliable source for these functions?  
I don't want to add to txAdmin a ctrl+c+v of some random code from the forum.  
That's how many fivem scripts get some badly optimized functions.

Reply:
Script was take from this: https://github.com/IllusiveTea/ScaleformAnnouncememts